selfdrive/locationd/paramsd.py

```diff
@@ -46,18 +46,25 @@ def __init__(self, CP, steer_ratio, stiffness_factor, angle_offset, P_initial=No
     self.yaw_rate_std = 0.0
     self.roll = 0.0
     self.steering_angle = 0.0
-    self.roll_valid = False
 
   def handle_log(self, t, which, msg):
     if which == 'livePose':
       device_pose = Pose.from_live_pose(msg)
       calibrated_pose = self.calibrator.build_calibrated_pose(device_pose)
-      self.yaw_rate, self.yaw_rate_std = calibrated_pose.angular_velocity.z, calibrated_pose.angular_velocity.z_std
+
+      yaw_rate_valid = msg.angularVelocityDevice.valid and self.calibrator.calib_valid
```

Comment:
@fredyshox can you make angularVelocityDevice invalid in a separate PR so it matches previous locationd.cc behavior?

Reply:
It already matches previous behavior